kahuna/app/lib/KahunaConfig.scala

```diff
@@ -60,5 +60,12 @@ class KahunaConfig(resources: GridConfigResources) extends CommonConfig(resource
     .getOrElse("This image can be used, but has warnings:")
   val unusableTextHeader: String = configuration.getOptional[String]("warningText.unusableHeader")
     .getOrElse("Unusable image")
+  val enableWarningFlags: Boolean = configuration.getOptional[Map[String, String]]("warningText.imagePreviewFlag").isDefined
```

Comment:
Would be useful to add these new configuration items in the [the config docs](https://github.com/guardian/grid/blob/main/docs/06-objects-of-interest/02-config.md)

Reply:
Hi @phillipbarron this PR updates the config docs https://github.com/guardian/grid/pull/3965